Tests/KeyboardManagerTests.swift

```diff
@@ -109,40 +164,38 @@ fileprivate extension KeyboardManagerTests {
 
     func postTestNotification(name: Notification.Name) {
         notificationCenter.post(name: name, object: nil, userInfo: [
-                UIKeyboardFrameEndUserInfoKey: NSValue(cgRect: endFrame),
-                UIKeyboardFrameBeginUserInfoKey: NSValue(cgRect: beginFrame),
-                UIKeyboardAnimationDurationUserInfoKey: NSNumber(floatLiteral: animationDuration),
-                UIKeyboardIsLocalUserInfoKey: NSNumber(booleanLiteral: isLocal),
-                UIKeyboardAnimationCurveUserInfoKey: curve
+            UIKeyboardFrameEndUserInfoKey: NSValue(cgRect: endFrame),
+            UIKeyboardFrameBeginUserInfoKey: NSValue(cgRect: beginFrame),
+            UIKeyboardAnimationDurationUserInfoKey: animationDuration,
+            UIKeyboardIsLocalUserInfoKey: isLocal,
+            UIKeyboardAnimationCurveUserInfoKey: curve,
         ])
     }
 
     func postWrongTestNotification() {
         notificationCenter.post(name: Notification.Name.UIKeyboardDidShow, object: nil, userInfo: [
-                UIKeyboardFrameEndUserInfoKey: NSValue(cgRect: endFrame),
-                UIKeyboardAnimationCurveUserInfoKey: 10
+            UIKeyboardFrameEndUserInfoKey: NSValue(cgRect: endFrame),
+            UIKeyboardAnimationCurveUserInfoKey: 10,
         ])
     }
 
-    func compareWithTestData(another data: KeyboardManagerEventData) -> Bool {
+    func compareWithTestData(another data: KeyboardManagerEvent.Data) -> Bool {
         let isFrameEqual = data.frame.begin == beginFrame &&
-                data.frame.end == endFrame
+            data.frame.end == endFrame
         return isFrameEqual &&
-                data.animationDuration == animationDuration &&
-                data.animationCurve == curve &&
-                data.isLocal == isLocal
+            data.animationDuration == animationDuration &&
+            data.animationCurve == curve &&
+            data.isLocal == isLocal
     }
 }
 
-extension KeyboardManagerEventData: Equatable {
-
+extension KeyboardManagerEvent.Data: Equatable {
 }
 
-public func ==(lhs: KeyboardManagerEventData, rhs: KeyboardManagerEventData) -> Bool {
+public func == (lhs: KeyboardManagerEvent.Data, rhs: KeyboardManagerEvent.Data) -> Bool {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/alphatroya/KeyboardManager/blob/25a2ec034b913c48e066e9db453abead1303374d/Tests/KeyboardManagerTests.swift#L195"></span>Public declarations should be documented.</td>
    </tr>
  </tbody>
</table>